pick.c

```diff
@@ -795,6 +807,7 @@ get_key(char *buf, size_t size, size_t *nread)
 	}	keys[] = {
 		KEY(ALT_ENTER,	"\033\n"),
 		KEY(ALT_ENTER,	"\033\r"),
+		KEY(BACKSPACE,	"\b"),
```

Comment:
OK, that will look "cleaner" - should "switch" in "selected_choice(void)" also sorted?